core-linux/src/main.cpp

```diff
@@ -98,14 +101,23 @@ int main(int argc, char **argv)
         int width = 800;
         int height = 600;
         int fullscreen = 0;
+        bool is_always_on_top = false;
+        std::string iconfile = "neutrolino.png";
         if(!options["window"].is_null()) {
             json windowProp = options["window"];
             width =  stoi(windowProp["width"].get<std::string>());
             height =  stoi(windowProp["height"].get<std::string>());
             if(!windowProp["fullscreen"].is_null())
                 fullscreen =  windowProp["fullscreen"].get<bool>() ? 1 : 0;
```

Comment:
@shalithasuranga Hi. May I ask something? Why use the number as boolean representation instead of actual boolean value itself?

Reply:
Ah sure.. that's because webview's method accepts int guess. So we just simply used int :)

Reply:
oh i see. thanks for the explanation